src/services/fetchdata/fetchdata.jsx

```diff
@@ -12,9 +11,22 @@ export const fetchRegionList = async () => {
   return data;
 };
 
-export const fetchPokemonByName = async (name) => {
-  const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${name}`);
+export const fetchPokemonByName = async (name, signal) => {
+  const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${name}`, {
+    signal: signal,
```

Comment:
I'm not sure that `signal` is a part of a fetch http request. I'm also not seeing it being passed into this function at any point

Reply:
yeah, i was trying something out to fix the memory leak, I forgot to delete it.